plugins/woocommerce/client/blocks/assets/js/blocks/payment-method-icons/edit.tsx

```diff
@@ -0,0 +1,119 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { useBlockProps, InspectorControls } from '@wordpress/block-editor';
+import { PanelBody, RangeControl } from '@wordpress/components';
+
+type BlockAttributes = {
+	numberOfIcons: number;
+};
+
+type PaymentMethod = {
+	icon: string;
+};
+
+type PaymentMethods = Record< string, PaymentMethod >;
+
+const CardPreview = ( {
+	type,
+	icon,
+}: {
+	type: string | undefined;
+	icon: string | undefined;
+} ) => {
+	let CardIcon = null;
+
+	if ( type && icon ) {
+		CardIcon = (
+			<div className="wp-block-woocommerce-payment-method-icons__item">
```

Comment:
We usually mark WooCommerce blocks with `wc` prefix - sorry I didn't spot that earlier!

```suggestion
			<div className="wc-block-payment-method-icons__item">
```